tests/conftest.py

```
@@ -1,10 +1,95 @@
+
+@pytest.fixture(scope="function")
+def reviewed_card_factory(db_manager, new_card_factory):
+    """
+    Factory fixture to create a card that has already been reviewed once.
+    This is useful for testing cards in states other than 'New'.
+    """
+
+    def _create_card(deck_uuid, content="Reviewed Card Content"):
+        card = new_card_factory(deck_uuid, content)
+        # Simulate a review to move it out of the 'New' state
+        card.state = CardState.LEARNING
```

Pylint (3.3.7)

[error] Class 'CardState' has no 'LEARNING' member

(E1101)